implementations/c/lib/linked_list/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+# Ockam queue
+
+project(queue C)
```

Comment:
Copy&paste artefact. Also, maybe we're doing something wrong here, if it still works with duplicate project names